js/data/bucket.js

```diff
@@ -65,8 +61,8 @@ function Bucket(options) {
     this.zoom = options.zoom;
     this.overscaling = options.overscaling;
     this.layer = options.layer;
+    this.childLayers = options.childLayers;
```

Comment:
Having `Bucket#layers` and `Bucket#layer` could be confusing. I'd be happy with any naming scheme that has a modifier on one of the two, such as `Bucket#parentLayer` and `Bucket#layers`.